src/app/components/primary-navbar/primary-navbar.component.ts

```diff
@@ -1,15 +1,80 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, HostListener, OnInit } from '@angular/core';
+import { AuthService } from 'app/core/auth.service';
+import { UserService } from 'app/core/user.service';
+import { NavbarDropdownService } from 'app/core/navBarDropdown.service';
+import { NavbarService } from '../../core/navbar.service';
+import { Topic } from '../../../entity';
+
 
 @Component({
   selector: 'clark-primary-navbar',
   templateUrl: './primary-navbar.component.html',
-  styleUrls: ['./primary-navbar.component.scss']
+  styleUrls: ['./primary-navbar.component.scss'],
+  providers: [NavbarDropdownService]
 })
 export class PrimaryNavbarComponent implements OnInit {
 
-  constructor() { }
+  showNav: boolean;
+  userDropdown: boolean;
+  isLoggedIn: boolean;
+  isDesktop: boolean;
+  isMSearch: boolean;
+  isMHamburger: boolean;
+  showTopics: boolean;
+  showResources: boolean;
+  topics: Topic[];
+  resizeThreshold = 1024;
+  externalResources: {name: string, link: string}[];
+  levelChoice: string;
+  // levelsDropdown: boolean;
```

Comment:
Is this comment being used? or will it be used for a refactor? If its not being used, it should be removed

Reply:
the levels dropdown was scrapped but we want to refactor it later for a different purpose, mike told me to just comment it all out

Reply:
Sounds good. I would just leave a comment next to it (and the rest) noting that so someone doesnt accidently delete them